dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.restfulWS3.0-xmlBinding3.0.feature

```diff
@@ -1,7 +1,7 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.restfulWS3.0-xmlBinding3.0
 visibility=private
-IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.restfulWSClient-3.0))", \
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.restfulWSClient-3.0)(osgi.identity=io.openliberty.restfulWSClient-3.1)))", \
  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.xmlBinding-3.0))"
```

Comment:
This PR should go in after https://github.com/OpenLiberty/open-liberty/pull/19534
That will make a `xmlBinding-4.0` feature should be available (even if it is still based on 3.0) and so this auto-feature should take that into account.